### ISSUE TITLE
dyson sphere rotation too fast. 

### ISSUE THREAD

**Issue by darkracer125** [Touhma/DSP_Galactic_Scale#235]
so i am running this with nebula multiplayer on a server.  and i was noticing the server was running slower and slower. 
untill i noticed there were an awefull lot of rockets on their way to the nodes of our dyson sphere. 

so because the star is bigger.  and the orbit of the outer shell is so big.  it's angular speed of 2.77 is too fast for the rockets to get to the nodes.  so they are just piling up.  
(rockets go 300.  shell is going 405)  

since it's a server i can't use the sphere editor tools to change it's angular speed. 
it would be nice if it defaulted to a lower speed. (i don't know if that's even possible?) 

or maybe someone knows of a mod that speeds up the rockets.  like doubles their speed.?

**Comment by innominata** [comment]
Hmm. You are saying sphere editor tools can change the angular speed?

**Comment by innominata** [comment]
talking to starfish now about possibly capping the max speed for sphere shells in GS2

**Comment by darkracer125** [comment]
yeah sphere editor tools can change the angular speed.   though seeing as i am running it as a headless server.  it's a bit difficult to do that seeing as the clients who join cannot make those changes. 
and the actual host only has the nebula console. 
but in single player this wouldn't even be a problem.  i'd slow down the sphere until it's constructed and then speed it back up again. 

i'd rather have faster rockets to be honest.  sadly there doesn't seem to be a mod for that.
it's actually quite cool to have a set of faster counter rotating rings in various axis. 
so the angular speed is only a problem during construction.  

but capping would deffinatly solve the problem. i just hope it isn't a hardcap so we can still speed it up with sphere editor tools.

**Comment by innominata** [comment]
From memory changing rocket speeds isn't trivial as they are computed on GPU. could be wrong
and any cap would be at galaxy generation time, or retroactively with a debug button

**Comment by innominata** [comment]
tho for server use a config var might be needed. I havent played with nebula at all

**Comment by darkracer125** [comment]
A config variable would work great

**Comment by AndrewLuebke** [comment]
Closing — 2.78.7 addressed both halves of this: new shell layers now get proper Kepler angular speeds (large-radius shells rotate slower), and rocket/bullet speeds scale with the sphere size so nodes can actually be caught. If rockets still pile up behind a shell on current (2.78.8), comment and we'll reopen — a config cap for existing over-speed shells would be the follow-up.
